test/test_dev_release.py

```diff
@@ -21,19 +21,35 @@ class MockOptions(object):
 class TestUntrusted(unittest.TestCase):
 
     def setUp(self):
+        apt.apt_pkg.config.set("Unattended-Upgrade::"
+                               "Skip-Updates-On-Metered-Connections",
+                               "false")
+        apt.apt_pkg.config.set("Unattended-Upgrade::OnlyOnAcPower",
+                               "false")
+        unattended_upgrade.LOCK_FILE = "./u-u.lock"
         self.rootdir = os.path.abspath("./root.untrusted")
         self.log = os.path.join(
             self.rootdir, "var", "log", "unattended-upgrades",
             "unattended-upgrades.log")
 
+        self.apt_conf = os.path.join(self.rootdir, "etc", "apt",
+                                     "apt.conf")
+
+        os.rename(self.apt_conf, self.apt_conf + ".bak")
```

Comment:
would not it be nicer to use a fragment in apt.conf.d/ ?

Reply:
I don't know, I feel like we should do some generic refactoring of the test rootdir interaction.

Reply:
well, i share the feeling then :-)
it is fine for now